apps/yaft/fb/common.cpp

```diff
@@ -58,8 +58,13 @@ draw_sixel(rmlib::fb::FrameBuffer& fb, int y_start, int col, uint8_t* pixmap) {
       src_offset = BYTES_PER_PIXEL * (h * CELL_WIDTH + w);
       memcpy(&color, pixmap + src_offset, BYTES_PER_PIXEL);
 
-      dst_offset = (y_start + h) * fb.canvas.lineSize() +
-                   (col * CELL_WIDTH + w) * fb.canvas.components();
+      if (term->isLandscape) {
```

Comment:
Seems to work, but doesn't have the margin applied.

Reply:
Ah, that's because the `(term->width - term->cols * CELL_WIDTH) / 2` part of `margin_left` isn't applied. cc @timower should I add the margin?

Reply:
Yeah, makes sense. I've never used sixel support much